### PR TITLE
Minor support page style fixes (ISSUE-282)

### LINKED ISSUE #282: Implement- Help/Support page
## Context
For the MVP, we need to add a help/support contact page which is accessible by all registered and logged in users.

## Process
On both the MAC user side and partner user side, we need to add a button in the header which directs them to Help.  This page should have a link to the relevant manual, as well as the email and instructions for how to ask for help.  Copy and current app images are listed below in References.

## Acceptance Criteria/Definition of Done

Step 1- @gujacq - please mock-up placements in both the MAC user and partner user headers for a Help page button.  Also please mockup a short Help page using the copy below in Resources.

Step 1- dev team- please implement the copy/mockups as listed below.

## Resources

MAC user header currently in app:
image

_MAC user help page copy:_
If you're having issues in the app, please check out our FAQ manual here first (LINK: https://docs.google.com/document/d/1GEq8hbnjmH_4xq1WFgvuVrLMSTU1knjHLrKZyVqKzcY/edit?usp=sharing)

If you aren't able to find the answer there, you can also email lockbox@midwestaccesscoalition.org

Partner user header currently in app:
image

_Partner user help page copy:_
If you're having issues in the app, please check out our FAQ manual here first (LINK: https://docs.google.com/document/d/1bhUsBvOoS8E15s8XPrQMejrZONrp-DuZJ42AjPf2odw/edit?usp=sharing)

If you aren't able to find the answer there, you can also email lockbox@midwestaccesscoalition.org

### PR DESCRIPTION
**Pull request by scottyschup**
## Changelog
* minor styling fixes for support page
* adds ability to change name for users
* add Google webfonts to application layout header
* refactor `DashboardController` to set `@lockbox_partner` with `before_action`

## Link to issue:  
Fixes #282 (finishes really, as Kevin already did most of that work)

#### Support page MOBILE
Screen Shot 2020-02-02 at 5 59 58 PM

#### Support page DESKTOP
Screen Shot 2020-02-02 at 6 08 42 PM

#### Support page, nav bar open MOBILE
Screen Shot 2020-02-02 at 6 00 07 PM

#### Support page, nav bar open DESKTOP
Screen Shot 2020-02-02 at 6 00 43 PM

#### Support page MOBILE
Screen Shot 2020-02-02 at 5 25 30 PM

#### Support page DESKTOP
Screen Shot 2020-02-02 at 5 25 12 PM

#### Support page, nav bar open MOBILE
Screen Shot 2020-02-02 at 5 56 55 PM

#### Support page, nav bar open DESKTOP
Screen Shot 2020-02-02 at 5 57 11 PM

## Are you ready for review?:
Hell yeah!

- [x] Added relevant tests
- [x] Linked PR to the issue
- [ ] Added notes for QA/special notes
- [x] Added relevant screenshots

### CODE REVIEW

File: app/controllers/dashboard_controller.rb [hunk 1]
```diff
@@ -1,16 +1,19 @@
 class DashboardController < ApplicationController
+  before_action :set_lockbox_partner
+
   def index
-    if current_user.partner?
-      @lockbox_partner = current_user.lockbox_partner
-    else
-      @lockbox_partners = LockboxPartner.all
-    end
+    @lockbox_partners = @lockbox_partner.nil? ? LockboxPartner.all : nil
```
Comment (BintLopez) [inline app/controllers/dashboard_controller.rb]: going to make this more role based and explicitly check for 'admin' role before displaying all Lockbox Partners

**Comment by BintLopez** [comment]
The addition of the users controller as a way to update the name has implications for this issue https://github.com/MidwestAccessCoalition/lockbox_rails/issues/324

File: app/views/layouts/_nav_bar.html.erb [hunk 2]
```diff
@@ -18,28 +25,45 @@
...
-        <% elsif lp = current_user.lockbox_partner %>
-          <h2><%= lp.name %></h2>
-          <p><%= lp.street_address %></p>
-          <p><%= lp.city %>, <%= lp.state %> <%= lp.zip_code %></p>
-          <p><%= lp.phone_number %></p>
-          <p><%= link_to "Edit Contact Information", edit_lockbox_partner_path(lp), class: "edit_contact_info btn btn-primary btn-invert" %></p>
-        <% end %>
+        <!-- Left container -->
+        <div class="navbar-subcontainer">
+          <% if current_user.admin? %>
```
Comment (BintLopez) [inline app/views/layouts/_nav_bar.html.erb]: all users could benefit from this functionality... and would resolve https://github.com/MidwestAccessCoalition/lockbox_rails/issues/324 cc/ @misternu

**Comment by BintLopez** [comment]
@scottyschup @misternu Added the ability for any user to change their own name. 
image

image

My test is currently failing but wanted to push these changes anyways before sunday

File: app/controllers/dashboard_controller.rb [hunk 3]
```diff
@@ -1,16 +1,19 @@
...
-      @lockbox_partner = current_user.lockbox_partner
-    else
-      @lockbox_partners = LockboxPartner.all
-    end
+    @lockbox_partners = current_user.admin? ? LockboxPartner.all : nil
   end
 
   def onboarding_success
-    @lockbox_partner = current_user.lockbox_partner
   end
```
Comment (nsiwnf) [inline app/controllers/dashboard_controller.rb]: Empty method?

Comment (kyaroch) [reply app/controllers/dashboard_controller.rb]: This is Rails Magic™. The controller will render the corresponding view even if you don't put anything in the method.

**Pull request merged**